### ISSUE TITLE
Holodeck plasma bug returned.

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#6371]
(WEB REPORT BY: yred REMOTE: 158.69.55.2:7777)
Revision 038130b573642fd5381de8c7ed5e1fb1db

Emagged holodeck plasma simulation produced ~2E12 kPa of plasma.

**Comment by Exxion** [comment]
I've seen someone cast the station into chaos with simple air by having pressures that high. Plasma would be horrifying.

**Comment by PJB3005** [comment]
I recall paradise having this exact issue (I think it was fixed but I can't find when/where), and if I recall correctly it happened by changing the holodeck too fast in succesion?

**Comment by clusterfack** [comment]
The bug is with the group multiplier of ZAS zones. I speak from experience on fixing a similar bug.

ZAS 'counts' a zone using a variable that holds the number of turfs within that zone. That variable easily breaks in certain circumstances, in the case that I fixed it was that turf/new was being called on top of a prior turf. This resulted in the group multiplier being incremented without the number of turfs changing, and therefore during the first gas exchange or zone change it created impossible amounts of gas from nothing.

TL;DR The holodeck must be breaking the group multiplier for it's ZAS zone (likely when it swaps areas)
